libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/accounts/SkuGenerator.kt

```diff
@@ -0,0 +1,7 @@
+package com.mapbox.services.android.navigation.v5.internal.accounts
+
+internal interface SkuGenerator {
+    fun generateSkuToken(): String
```

Comment:
Minor thing: `generateToken` as it's a method from `SkuGenerator` (it's explicitly a _Sku_ token), what do you think?